library/HTMLPurifier/ConfigSchema/schema/Filter.ExtractStyleBlocks.txt

```diff
@@ -54,7 +54,7 @@ EXTERNAL: CSSTidy
   <div>
     <?php echo $html; ?>
   </div>
-</b]]><![CDATA[ody>
+</b]]><![CDATA[body>
```

Comment:
this is wrong, it should be ody here!

Reply:
Apologies, thank you for addressing this and merging it in.